backend-api/src/functions/asyncActiveSession/tokenService/tests/tokenService.test.ts

```diff
@@ -24,355 +23,16 @@ describe("Token Service", () => {
       publicKeyGetter: () => new MockPubicKeyGetterGetPublicKeySuccess(),
     };
     tokenService = new TokenService(dependencies);
-    mockFetch = jest.spyOn(global, "fetch").mockImplementation(() =>
```

Comment:
No longer required - these tests are now part of publicKeyGetter.test.ts and sendHttpRequest.test.ts